src/client/src/app/app-routing.module.ts

```diff
@@ -19,6 +20,7 @@ const routes: Routes = [
   {path: 'games', loadChildren:
   () => import('./modules/games/module-games.module').then(m => m.GamesModule)
   },
+  {path: 'merch-details', component: MerchDetailsComponent},
```

Comment:
i might also try adding a this in the routes array in alphabetical order, or next to like routes for organization